.travis.yml

```diff
@@ -23,5 +23,5 @@ jobs:
       # Use codespell for spell checking
       install:
         pip install codespell
-      script: codespell --skip="assets,*.svg,bin" --quiet-level=2  --ignore-words-list="rouge,dropse"
+      script: codespell --skip="assets,*.svg,bin" --quiet-level=2  --ignore-words-list="rouge,dropse" -L "namd"
```

Comment:
Looks like `ignore-words-list` option has been replaced by `-L`, can you please update this line to reflect that:
```
script: codespell --skip="assets,*.svg,bin" --quiet-level=2  -L "namd","rouge","dropse"
```
(Either they are accepting the original option in deprecated mode, or the words no longer kick an error)